docs/Changelog.md

```diff
@@ -600,7 +600,7 @@ opset_import {
 <dd>Which axis to concat on</dd>
 </dl>
 
-#### Inputs (1 - &#8734;)
+#### Inputs (0 - &#8734;)
```

Comment:
so we need some annotation to signify that this variadic argument actually needs >=1 input. Shall we have Variadic and VariadicOptional?

basically your docs shouldn't change (it's a sign that the spec didn't change)

Reply:
This is really a great question!!! In my current implementation, variadic argument needs >= 0 input.

I was thinking of using bitwise options. Say Single=1, Optional=2, Variadic=4 (Here variadic needs >=1 input). However, I realized that only Optional and Variadic could be merged. So I didn't pick up the design and change variadic needs >=0 input (which means VariadicOptional in your idea).

What do you think?
1) Keep current design to have Variadic needs >=0 input. or,
2) Make Variadic needs >=1 input and add VariadicOptional needs >=0 input.

Reply:
btw, @dzhulgakov , I put the comment in schema.h to show the meaning of each option as.
  // Formal parameter options.

  enum FormalParameterOption {

    // The input formal parameter is single and not optional.

    // Number of this input is 1.

    Single = 0,

    // The input formal parameter is single and optional.

    // Number of this input is 0 or 1.

    Optional = 1,

    // The input formal parameter is variadic.

    // Number of this input is [0, n].

    Variadic = 2,

  };

I can put it in *.md documents also. Let's make a decision on the two solutions above firstly.